scripts/configure-cluster/common/setup-operators.sh

```diff
@@ -28,6 +28,8 @@ install_service_binding_operator(){
     name: $3
     source: $4
     sourceNamespace: $5
+    startingCSV: service-binding-operator.v0.8.0
```

Comment:
I am not sure if this version works properly on 4.7 cluster. Suspecting by seeing the logs. Earlier we used to run on `startingCSV: service-binding-operator.v0.3.0`, you can try this if it helps for kubernetes and ocp both.

Reply:
v0.3.0 is really old! It was released [in October 2020](https://github.com/redhat-developer/service-binding-operator/releases/tag/v0.3.0). Why are we using that? We removed the `startingCSV` from our script so that our CI would pick the latest available version of SBO each time.